src-electron/db/query-attribute.js

```diff
@@ -1120,17 +1121,21 @@ AND
   return rows.map(dbMapping.map.endpointTypeAttributeExtended)
 }
 
-exports.selectAllAttributeDetailsFromEnabledClusters =
+exports.selectAllAttributeDetailsFromEnabledClusters = dbCache.cacheQuery(
```

Comment:
These are attributes based on user selection. I am worried about this because this is dynamic data based on user selection in the UI. So it is not really a one time cache? How is this refreshed once user enables a new attribute?

Reply:
Does this affect performance when using the ZAP UI?